src/app/shared/data-resource/data-resource-detail/data-resource-detail.component.html

```diff
@@ -1,17 +1,88 @@
-<data-resource-data [resource]="resource"
-                    [dataResource]="dataResource"
-                    [resourceType]="resourceType"
+<document-detail-header class="d-block mb-4"
+                        [resource]="resource"
+                        [dataResource]="dataResource"
+                        [resourceType]="resourceType"
+                        [constraintData]="constraintData$ | async"
+                        [permissions]="permissions"
+                        [workspace]="workspace$ | async"
+                        (switchToTable)="onSwitchToTable()"
+                        (remove)="onRemove()"
+                        (versionClick)="selectedTab = detailTabType.Activity">
```

Comment:
Using simple property as input in another component can lead to unexpected behavior.